packages/unimodules-permissions-interface/src/PermissionsInterface.ts

```diff
@@ -11,4 +11,5 @@ export interface PermissionResponse {
   expires: PermissionExpiration;
   granted: boolean;
   canAskAgain: boolean;
+  scope?: 'whenInUse' | 'always' | 'none';
```

Comment:
I'm not fun of adding the `scope` key here. I think we should add it to the `Permissions.types.ts` and the `expo-localizations` package.

Reply:
Agrred, partially. Moved to expo-permissions.

Reply:
It needs to change this schema too - > https://github.com/expo/expo/blob/04e98becc3e7f8a8a1a99eb7f384da98bc0d3762/packages/expo-location/src/Location.types.ts#L210-L221